lib/_io.scm

```diff
@@ -10570,19 +10570,20 @@
 (define-prim (##readtable-setup-for-standard-level! rt)
   (let ((standard-level (##get-standard-level)))
 
-    (cond ((or (##fx= standard-level 4)  ;; R4RS language
+    (cond ((##fxzero? standard-level) ;; Gambit language, explicit
```

Comment:
Should keep the same style `(##fx= standard-level 0)`.

Reply:
Noted and fixed.